### PR TITLE
Dockerize Application

### PR DESCRIPTION
**Pull request by etagwerker**
**Description:**

Hi there,

This PR solves #70 and adds support for Docker and Docker Compose for both development and test environments.

Please check it out and let me know if it looks okay. Also, it would be great if you can make sure that it works. After following the setup instructions, try running this within your docker container:

```
rspec spec
```

I will abide by the [code of conduct](code_of_conduct.md).

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,18 @@ Pecas is a time tracking leaderboard for
...
     git clone git@github.com:fastruby/pecas.git
     cd path/to/pecas
-
-Copy the database.yml config:
-
+    docker-compose build
+    docker-compose run web /bin/bash
     ./bin/setup
 
 You must setup your `NOKO_TOKEN` in the `.env` file. You can setup your
```
Comment (cleicar) [inline README.md:27]: Should we have the link to where to generate the `NOKO_TOKEN`?

**Review by cleicar — APPROVED** [review]
This looks good @etagwerker. 

I was able to follow the steps in my machine. 

I have just two suggestions.

File: README.md (hunk 1 above)
Comment (etagwerker) [reply README.md:27]: @cleicar I think people will be able to figure it out. 👍

File: README.md [hunk 2]
```diff
@@ -10,13 +10,18 @@ Pecas is a time tracking leaderboard for
 
 To install Pecas in a development environment, you can follow the next steps:
 
+### Requirements
+
+- Docker
+- Docker-Compose
+- Git
+
```
[the remark this answers is not in the document]
Comment (etagwerker) [reply README.md:18]: @cleicar That's a good idea, although they will get it installed if they are using Docker + Docker Compose.

**Pull request merged**